core/cmd/client.go

```diff
@@ -281,15 +281,15 @@ func (s *server) run(port uint16, writeTimeout time.Duration) error {
 	s.lggr.Infof("Listening and serving HTTP on port %d", port)
 	s.httpServer = createServer(s.handler, port, writeTimeout)
 	err := s.httpServer.ListenAndServe()
-	s.lggr.ErrorIf(err, "Error starting server")
+	s.lggr.ErrorIf(err, "Error starting server, to disable the plaintext HTTP server set CHAINLINK_PORT=0")
```

Comment:
What about moving these messages below to the wrapped error, so that #5939 logs them as critical from the caller?